WebContent/js/components/JobInstance.js

```diff
@@ -131,6 +133,32 @@ class JobInstance extends React.Component {
         }
     }
 
+    handleDownloadJCL(job) {
```

Comment:
This is a copy of existing function from JobFile and file.id changed to JCL. This is not a good practice, I suggest to use existing function downloadJobFile() and change the way how the function is called.